### ISSUE TITLE
weather and invest applets no longer available in "add to panel" mate-panel-1.8.1

### ISSUE THREAD

**Issue by stevesss** [mate-desktop/mate-panel#284]
I used the source code mate-panel-1.8.1 to reinstall the mate-panel for mint-mate1.8.1. I had the the weather and invest applets installed from the binary version that came from the repositories. Now they don't work. When I refresh the panel, I get the following messages. Any help?
![invest](https://cloud.githubusercontent.com/assets/11133846/6313847/8431daec-b98d-11e4-945b-97f6b289d2ce.png)
![matepanel](https://cloud.githubusercontent.com/assets/11133846/6313848/84330d22-b98d-11e4-870d-fd130c970b6f.png)

**Comment by stevesss** [comment]
OK. I'm not a programmer so I apologize if this was an inappropriate post. I was able to solve my problem by downloading the source code for mate-applet and the compiling it and installing it. The invest applet still didn't work so I went into the invest sub folder and compiled it and installed it from there and viola everything fixed. Good lesson for me. Glad to know this resource is available when the code in the repositories doesn't work right.
